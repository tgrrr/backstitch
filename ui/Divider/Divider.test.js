import React from 'react'
import renderer from 'react-test-renderer'
import { Divider } from '.'

it('Divider: default', () => {
    const component = renderer.create(
        <Divider>
            <>
                Divider
            </>
        </Divider>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
