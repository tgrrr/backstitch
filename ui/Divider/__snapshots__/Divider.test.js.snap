// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Divider "
    data-testid="Divider"
  >
    <hr
      className="MuiDivider-root"
    >
      Divider
    </hr>
  </div>
</div>
`;
